content/foundations/css-utilities/borders.mdx

```diff
@@ -63,22 +63,10 @@ You can also add rounded corners to each edge (top, right, bottom, left) with th
   height="240"
 />
 
-## Responsive borders
+## Responsive rounded corners
 
-You can adjust border widths on all sides or each side individually with responsive border utilities:
+You can adjust border radius on all sides or each side individually with responsive border radius utilities:
 
-- `border-(sm|md|lg|xl)` adds borders on all sides at and above the breakpoint. The `border-(sm|md|lg|xl)` shorthand is also supported.
-- `border-(sm|md|lg|xl)-0` removes borders from all sides at and above the breakpoint.
-- `border-(sm|md|lg|xl)-(top|right|bottom|left)` adds a border on the given side at and above the breakpoint.
-- `border-(sm|md|lg|xl)-(top|right|bottom|left)-0` the border from the given side at and above the breakpoint.
-
-<StorybookEmbed
-  framework="css"
-  componentId="utilities-border"
-  stories={{'responsive': {name: 'Responsive'}}}
-  height="150"
-/>
-
-## Border colors
-
-Border colors are documented on the [colors utilities page](/foundations/css-utilities/colors#border).
+- `rounded-(sm|md|lg|xl)` adds a 6px rounded corner on all sides at and above the breakpoint.
+- `rounded-(sm|md|lg|xl)-(0|1|2|3|4)` adds a rounded corner on all sides at and above the breakpoint.
+- `rounded-(sm|md|lg|xl)-(top|right|bottom|left)-(0|1|2|3|4)` adds a rounded corner on the given side at and above the breakpoint.
```

Comment:
I guess we wanna wait until https://github.com/primer/css/pull/2386 is merged and then add a `<StorybookEmbed>` too?

Reply:
yep! I'll merge it now and add the `StorybookEmbed` component 👍